src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -571,6 +575,46 @@ public static List<ClusterMetadata> tcmLoadRange(long min, long max)
         return afterLoad;
     }
 
+    /**
+     * This method exists due to the fact that we define a retry policy for TCM to follow, and then TCM ignores it and does no retries...
+     */
+    private static List<ClusterMetadata> reconstruct(long min, long max)
```

Comment:
> 5) When accord asks TCM for the epoch history, there were no retries which would cause stability issues during startup

We define a retry policy but TCM doesn't use it, so need to do it ourselves.  I tried to figure out how to implement but I struggled so figured it was quick to do this in the caller for now...